core/src/main/scala/org/apache/spark/storage/BlockManagerMasterActor.scala

```diff
@@ -52,11 +52,7 @@ class BlockManagerMasterActor(val isLocal: Boolean, conf: SparkConf, listenerBus
 
   private val akkaTimeout = AkkaUtils.askTimeout(conf)
 
-  val slaveTimeout = {
-    val defaultMs = math.max(conf.getInt("spark.executor.heartbeatInterval", 10000) * 3, 45000)
-    val networkTimeout = conf.getInt("spark.network.timeout", defaultMs / 1000)
-    conf.getLong("spark.storage.blockManagerSlaveTimeoutMs", networkTimeout * 1000)
-  }
+  val slaveTimeout = conf.getLong("spark.storage.blockManagerSlaveTimeoutMs", 120 * 1000)
```

Comment:
if user do not set blockManagerSlaveTimeoutMs, i think we need to set spark.network.timeout to blockManagerSlaveTimeoutMs. if spark.network.timeout is not defined, we can set the default 120 to spark.network.timeout.